inference-engine/src/transformations/src/transformations/op_conversions/convert_mvn1.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_mvn1.hpp"
+
+#include <ngraph/rt_info.hpp>
+
+#include <numeric>
+
+#include <ngraph/opsets/opset2.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertMVN1, "ConvertMVN1", 0);
+
+ngraph::pass::ConvertMVN1::ConvertMVN1() {
+    auto mvn = pattern::wrap_type<opset2::MVN>(); // MVN was missing in opset1
+
+    ngraph::matcher_pass_callback callback = [](pattern::Matcher& m) {
+        auto mvn_node = std::dynamic_pointer_cast<opset2::MVN>(m.get_match_root());
+        if (!mvn_node) {
+            return false;
+        }
+
+        const auto input = mvn_node->input_value(0);
+
+        // MVN-1 support only 4D and higher input tensors
+        auto input_rank = input.get_partial_shape().rank();
+        if (input_rank.is_static() && input_rank.get_length() >= 4) {
```

Comment:
may be 
if( !(input_rank.is_static() && input_rank.get_length() >= 4)) 
{
return false;
}
...true branch...